src/main.js

```diff
@@ -1,4 +1,10 @@
 /* @flow */
 import {main} from './program';
+import cmd from './cmd';
+import * as logger from './util/logger';
 
-export {main};
+// This only exposes util/logger so far.
+// Do we need anything else?
```

Comment:
@kumar303 :+1: Nice! This is pretty neat!

The change seems ok to me, but I'm wondering if we should make some tweak to the defaultReloadStrategy, mostly to make the caller of the API able to disable or customize the "interactive" behavior (e.g. reading from stdin when used as a library doesn't seem a great idea, it is probably something that the caller may want to easily disable and maybe manage with an API object which allow it to know when a change has been detected, trigger the addon reload and exit the runner).

What do you think?